pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml

```diff
@@ -441,4 +433,3 @@ theme: pdf
       %p Launched in 2013, Code.org® is a non-profit dedicated to expanding access to computer science, and increasing participation by women and underrepresented students of color. Our vision is that every student in every school should have the opportunity to learn computer science.
     %p.clear-both.sources Data is from the Conference Board for job demand, the Bureau of Labor Statistics for state salary and national job projections data, the College Board for AP exam data, the National Center for Education Statistics for university graduate data, the Gallup and Google research study Education Trends in the State of Computer Science in U.S. K-12 Schools for schools that offer computer science and parent demand, and Code.org for its own courses, professional learning programs, and participation data.
```

Comment:
nit: We could just wrap these ourselves and the browser should reconstruct the lines just fine.